code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -1617,3 +1617,55 @@ GLOBAL_LIST_EMPTY(bloodmen_list)
 		H.blood_volume -= 35
 	spawn_atom_to_turf(/mob/living/simple_animal/hostile/asteroid/hivelordbrood/bloodling, owner, 3, TRUE) //think 1 in 4 is a good chance of not being targeted by fauna
 	next_expulsion = world.time + cooldown
+
+// CS Knife
+
+/obj/item/cs_knife
+	name = "M9 Bayonet"
+	desc = "A simple knife, one edge serraded and the other deadly sharp. It fills you with a compulsion to 'Rush B', whatever that means."
+	lefthand_file = 'yogstation/icons/mob/inhands/weapons/cs_knife_base_lefthand.dmi'
+	righthand_file = 'yogstation/icons/mob/inhands/weapons/cs_knife_base_righthand.dmi'
+	icon = 'yogstation/icons/obj/lavaland/artefacts.dmi'
+	icon_state = "cs_knife_base"
+	force = 10
+	damtype = BRUTE
+	sharpness = SHARP_EDGED
+	w_class = WEIGHT_CLASS_SMALL
+	hitsound = 'yogstation/sound/weapons/csknife.ogg'
+	attack_verb = list("slashed","stabbed")
+	var/fauna_backstab_bonus = 50
+	var/person_backstab_bonus = 10
+	item_flags = SLOWS_WHILE_IN_HAND
+	unique_reskin = list("Default" = "cs_knife_base",
+						"Marble Fade" = "cs_knife_marble",
+						"Ultraviolet" = "cs_knife_uv",
+						"Autotronic" = "cs_knife_auto",
+						"Doppler Saphine" = "cs_knife_ds",
+						"Doppler Ruby" = "cs_knife_dr",
+						"Doppler Emerald" = "cs_knife_de"
+						)
+
+/obj/item/cs_knife/Initialize()
+	. = ..()
+	slowdown = -0.6
+
+/obj/item/cs_knife/afterattack(atom/target, mob/living/user, proximity_flag, clickparams)
+	. = ..()
+	if(proximity_flag && isanimal(target))
+		var/mob/living/L = target
+		if(!QDELETED(L))
+			new /obj/effect/temp_visual/kinetic_blast(get_turf(L))
+			var/backstab_dir = get_dir(user, L)
+			var/def_check = L.getarmor(type = MELEE)
+			if((user.dir & backstab_dir) && (L.dir & backstab_dir))
+				L.apply_damage(force + fauna_backstab_bonus, BRUTE, blocked = def_check)
+				playsound(user, 'yogstation/sound/weapons/csknife_backstab.ogg', 100, 1)
```

Comment:
this will apply both the backstab and the normal stab sound. if you want to prevent this you could move a lot of this code to pre-attack and set the attack sound before the attack is made, only to revert to initial value after attack.